sdks/python/apache_beam/ml/inference/vertex_ai_inference.py

```diff
@@ -41,6 +41,22 @@
 # pylint: disable=line-too-long
 
 
+def retry_on_gcp_client_error(exception):
```

Comment:
nit: prepend with _ as this is an internal helper. Alternatively, could also inline this function via a `lambda` & include the rationale in the PR (no strong opinion here).

Reply:
Done